arches/app/media/js/views/components/workflows/final-step.js

```diff
@@ -6,15 +6,7 @@ define([
         this.urls = arches.urls;
         this.loading = ko.observable(true);
         this.workflows = ko.observableArray();
-        try {
-            this.resourceid = ko.unwrap(params.workflow.resourceId);
-        } catch(e) {
-            try {
-                this.resourceid = ko.unwrap(params.form.resourceId);
-            } catch(e) {
-                // pass
-            }
-        }
+        this.resourceid = params.resourceid;
```

Comment:
This change seems to break some of the afs workflows.

Reply:
We'll have to adjust the AfS steps - maybe after I finish this ticket: https://github.com/archesproject/arches/issues/7983